delete.go

```diff
@@ -43,21 +61,17 @@ status of "ubuntu01" as "stopped" the following will delete resources held for
 		if err != nil {
 			return err
 		}
-		switch s {
-		case libcontainer.Stopped:
-			destroy(container)
-		case libcontainer.Created:
-			container.Signal(syscall.SIGKILL)
-			for i := 0; i < 100; i++ {
-				time.Sleep(100 * time.Millisecond)
-				if err := container.Signal(syscall.Signal(0)); err != nil {
-					destroy(container)
-					return nil
-				}
+		if context.Bool("force") {
+			killContainer(container)
```

Comment:
Looks weird to send signal to a stopped container, though it won't cause any errors.